bridges/JornalDeNoticiasBridge.php

```diff
@@ -0,0 +1,56 @@
+<?php
+class JornalDeNoticiasBridge extends BridgeAbstract {
+	const NAME = 'Jornal de Notícias (PT)';
+	const URI = 'https://jn.pt';
+	const DESCRIPTION = 'Jornal de Notícias (JN.PT)';
+	const MAINTAINER = 'somini';
+	const PARAMETERS = array(
+		'URL' => array(
+			'url' => array(
+				'name' => 'URL (relative)',
+				'exampleValue' => 'opiniao/catia-domingues.html',
+			)
+		)
+	);
+
+	public function getIcon() {
+		return 'https://static.globalnoticias.pt/jn/common/images/favicons/favicon-128.png';
+	}
+
+	public function getURI() {
+		switch($this->queriedContext) {
+		case 'URL':
+			$url = self::URI . '/' . $this->getInput('url');
+			break;
+		default:
+			$url = self::URI;
+		}
+		return $url;
+	}
+
+	public function collectData() {
+		$archives = self::getURI();
+		$html = getSimpleHTMLDOMCached($archives)
+			or returnServerError('Could not load content');
```

Comment:
You don't need "or returnServerError('Could not load content');". getSimpleHTMLDomCache will raise exception on non http 200 return code.